### ISSUE TITLE
[NuGet.org Bug]: Anchor link focus outline makes it hard to read

### ISSUE THREAD

**Issue by pranavkm** [NuGet/NuGetGallery#8994]
### Impact

It bothers me. A fix would be nice

### Describe the bug

On NuGet.org, focusing on an anchor link adds a fairly thick outline which makes it hard to read the anchor text. For instance, I tabbed on the version listing for `Microsoft.Extensions.DependencyInjection` and here's what that looks like:

![image](https://user-images.githubusercontent.com/174281/153080150-fd3cf34b-e619-48b7-b94e-17e2ea54ee2a.png)

It'd be nice not to have the text cut-off when focused like so.

### Repro Steps

1. Visit https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/
2. Keep tabbing until you get to text that is small

### Expected Behavior

Legible text when focused.

**Comment by ryuyu** [comment]
I believe we are within spec, but this is still a problem.
I'll see if we can make the stroke for focus go around the outside.
I'm guessing we are using an inner stroke AND outer stroke for focus (at least that is what it looks like)
We should be able to work around this by moving to an outer stroke only and doubling the thickness.
In any case, I will look in to this.
